### PR TITLE
Exclude TS files in NPM exports

### PR DESCRIPTION
**Pull request by AbhilashSrivastava**

### CODE REVIEW

**Comment by tusharmath** [comment]
Can you update the git commit message to be of the format — `fix(package): ignore .ts files from npm`

**Comment by AbhilashSrivastava** [comment]
Sure Tushar

File: .npmignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 !*.js
 !*.map
 !.d.ts
```
Comment (f0rr0) [inline .npmignore:5]: should be `!*.d.ts`?

Comment (tusharmath) [reply .npmignore:5]: Yes. Correct.

**Comment by tusharmath** [comment]
duplicate #14

**Pull request closed without merging**